src/lf_mask.rs

```diff
@@ -449,15 +450,17 @@ pub(crate) fn rav1d_create_lf_mask_intra(
     let bx4 = bx & 31;
     let by4 = by & 31;
 
+    let filter_level_yuv = filter_level.0.map(|a| a[0][0]);
```

Comment:
@kkysen could you make the same changes to `rav1d_create_lf_mask_inter`? That's the function that's actually hot.

I don't think we'll see a significant improvement from this alone without the loop unrolling but it doesn't hurt to merge.